### ISSUE TITLE
Inquiring about Interest in dynamically changing link properties through seedemu-client map view

### ISSUE THREAD

**Issue by Bruol** [seed-labs/seed-emulator#217]
Hello,

As part of Project I am working on with ETH Zurich I am working on some parts of the seed-emulator.

In the project we are interested in the possibility to change link properties such as bandwidth, latency, loss and jitter dynamically while the docker containers are running. We thought about the possibility to add this functionality to the map view of the seedemu-client. 

**In this issue I wanted to ask whether this is something that you would be interest in adding to map view upstream. Or whether this does not align with what you have in mind with the map view.**

The link property controls could be added to the Details pane in the map view.

<img width="323" alt="image" src="https://github.com/seed-labs/seed-emulator/assets/38534927/8d7b3aaf-2769-4256-86cc-aca9e3c5c9ea">

This functionality could be achieved through executing tc commands in the router containers from the seedemu-client container

**Comment by kevin-w-du** [comment]
Yes, we are very interested in this feature. Actually, we have been using the tc command to set the link properties from the command line. If you can add this control to the map, that will be a very good contribution.  Thanks for the help.
